paperclip.gemspec

```diff
@@ -18,15 +18,15 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.requirements << "ImageMagick"
-  s.required_ruby_version = ">= 1.9.2"
+  s.required_ruby_version = ">= 2.0.0"
 
-  s.add_dependency('activemodel', '>= 3.0.0')
-  s.add_dependency('activesupport', '>= 3.0.0')
+  s.add_dependency('activemodel', '>= 3.2.0', '>= 4.1.0')
```

Comment:
Seems these should be `"~>3.2.0", "~>4.1"` right? Otherwise `">= 3.2.0"` will still cover Rails 4.0.